include/ck_tile/core/algorithm/static_encoding_pattern.hpp

```diff
@@ -56,21 +56,29 @@ template <index_t BlockSize,
           index_t YPerTile,
           index_t XPerTile,
           index_t VecSize,
+          index_t NumWaveGroups,
           tile_distribution_pattern DistributionPattern>
 struct TileDistributionEncodingPattern2D : public TileDistributionEncodingPattern
 {
 };
 
 // Thread raked
-template <index_t BlockSize, index_t YPerTile, index_t XPerTile, index_t VecSize>
+template <index_t BlockSize,
+          index_t YPerTile,
+          index_t XPerTile,
+          index_t VecSize,
+          index_t NumWaveGroups>
 struct TileDistributionEncodingPattern2D<BlockSize,
                                          YPerTile,
                                          XPerTile,
                                          VecSize,
+                                         NumWaveGroups,
                                          tile_distribution_pattern::thread_raked>
     : public TileDistributionEncodingPattern
 {
 
+    static_assert(NumWaveGroups == 2);
```

Comment:
Please don't do that. You're now changing behavior of other kernels using this!

Reply:
Removed.